typhos/panel.py

```diff
@@ -266,6 +266,36 @@ def _create_row_label(self, attr, dotted_name, tooltip, long_name=None):
             label.setToolTip(_tooltip)
         return label
 
+    def _get_long_name(self, device, attr, dotted_name) -> str:
+        """
+        Check the signal for its long_name, if it exists. 
+        Until Ophyd makes it a standard signal, need to manually check
+        the device and its components for the name.
+
+        Parameters
+        -----------
+            device: (any)
+                The Ophyd.Device with component signals
+            attr: (str)
+                The str name of the signal attribute
+            dotted_name: (str)
+                The full dotted name to the signal
+
+        Returns
+        --------
+            str or None
```

Comment:
Super petty spacing nitpick to match the other docstrings
```suggestion
        device: (any)
            The Ophyd.Device with component signals
        attr: (str)
            The str name of the signal attribute
        dotted_name: (str)
            The full dotted name to the signal

        Returns
        --------
        str or None
```

Reply:
I don't think this ever got committed 👀

Reply:
whoops

Reply:
I don't even remember getting this message on the commit o.O